build.assets/tooling/cmd/please-run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
+
+# This script is a helper that tells developers what generated content is out of date
+# and which command to run.
+# When running on GitHub actions, the script will also create an error in the PR and
+# collapse the diff to improve readability.
+
+set -eu
+
+KIND="$1"
+GENERATE_COMMAND="$2"
+
+TITLE="$KIND are out-of-date"
+MESSAGE="Please run the command \`$GENERATE_COMMAND\`"
```

Comment:
Note that, if we keep must-start-clean/host, then it's possible not all callers have a GENERATE_COMMAND.

Reply:
I don't think we should to keep `must-start-clean/host` as it should never fail in the CI, and this usually means something is going very wrong and the dev can't fix it by themselves. Ci should always start clean. I'll remove it.